src/server/constants.py

```diff
@@ -53,3 +53,5 @@ class QUERIES:
                                         WHERE date BETWEEN %s AND %s
                                         GROUP BY period_number
                                         ORDER BY period_number"""
+
+    WALLET_DEPOSIT: str = "UPDATE users SET wallet_usd = wallet_usd + %s WHERE uuid=%s"
```

Comment:
Also, since you are adding a new query, can you change `str` types of all queries listed here to `psycopg.abc.Query`?

```python
from psycopg.abc import Query
```